classes/Task.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+// Класс, представляющий сущность "Задание"
+class Task
+{
+    private int $customer_id;
+    private ?int $executor_id;
+    private string $current_status = self::STATUS_NEW;
+
+    const STATUS_NEW = 'status_new';
+    const STATUS_CANCELED = 'status_canceled';
+    const STATUS_IN_WORK = 'status_in_work';
+    const STATUS_PERFORMED = 'status_performed';
+    const STATUS_FAILED = 'status_failed';
+    const ACTION_CANCEL = 'action_cancel';
+    const ACTION_RESPOND = 'action_respond';
+    const ACTION_EXECUTE = 'action_execute';
+    const ACTION_REFUSE = 'action_refuse';
+
+    public function __construct(int $customer_id, int $executor_id = null)
+    {
+        $this->executor_id = $executor_id;
+        $this->customer_id = $customer_id;
+    }
```

Comment:
Если версия PHP 8+, то конструктор лучше переписать. Вместо того, чтобы объявлять у класса свойства, а потом присваивать им значения через конструктор, в версиях 8+ есть сокращённая запись. При такой записи объявление этих свойств в 6 и 7 строках надо убрать. Плюс, если значение свойства не будет меняться по мере выполнение скрипта, то можно ещё указать директиву `readonly`
```suggestion
    public function __construct(
        private readonly int $customerId,
        private int $executorId = null
    ) {
    }
```